microdnf/context.cpp

```diff
@@ -176,6 +179,47 @@ std::chrono::time_point<std::chrono::steady_clock> MicrodnfRepoCB::prev_print_ti
 
 }  // namespace
 
+void Context::set_cache_dir() {
```

Comment:
I think we'll want to move this and the following method's logic into libdnf, right? If so, can you add a TODO for it?

Reply:
Yes, the goal is to isolate the logic that will be moved to libdnf. I added TODO.